styleguide/_static/issue-templates/documentation.md

```diff
@@ -0,0 +1,37 @@
+---
+name: 📚 Documentation 
+about: Add/Improve Documentation
+title: 📚 <Insert Title>
+labels: documentation
+assignees: ''
+
+---
+
+# Summary
+<!---
+A short summary on what of what needs to be documented or updated.
+--->
+
+# Details
+<!---
+A detailed description about what kind of documentation should be added and/or updated and why.
+--->
+
+## Background & Context
+<!--- Optional --->
```

Comment:
The reasons why things should be done are usually very important. Here it is optional, for Feature not. I am thinking, about if should not make it optional here too